packages/webcert/src/feature/certificate/Inputs/UeUncertainDate.tsx

```diff
@@ -98,7 +109,7 @@ const UeUncertainDate: React.FC<Props> = ({ question, disabled }) => {
       </div>
       <div className="iu-width-xxl">
         <label htmlFor={'day_' + question.id}>Dag</label>
-        <input id={'day_' + question.id} type="text" disabled={true} value="00" className="ic-textfield iu-color-muted iu-border-muted" />
+        <TextInput id={'day_' + question.id} onChange={() => null} disabled={true} value="00" css={styleTextField} />
```

Comment:
Instead of having a placeholder function for `onChange` we could change the `onChange` parameter to be optional in `TextInput` component. We could also pass on `hasValidationError={hasValidationError}`  here as well.